### PR TITLE
docs: fix feature testing sample code

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
Fix feature testing sample code.

**Checklist:**
- [x] Securely signed commits
- [n/a] Component(s) with PHPdocs
- [n/a] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/testing/feature.rst [hunk 1]
```diff
@@ -379,7 +379,7 @@ Asserts that $fragment is found within the JSON response. It does not need to ma
     ]
 
     // Is true
-    $this->assertJSONFragment(['config' => ['key-a']);
+    $results->assertJSONFragment(['config' => ['key-a']]);
```
Comment (MGatner) [inline user_guide_src/source/testing/feature.rst:382]:
```suggestion
    $result->assertJSONFragment(['config' => ['key-a']]);
```
Can you make this "result" (singular) to be consistent?

File: user_guide_src/source/testing/feature.rst [hunk 2]
```diff
@@ -379,7 +379,7 @@ Asserts that $fragment is found within the JSON response. It does not need to ma
...
 
 .. note:: This simply uses PHPUnit's own `assertArraySubset() <https://phpunit.readthedocs.io/en/7.2/assertions.html#assertarraysubset>`_
```
Comment (MGatner) [inline user_guide_src/source/testing/feature.rst:384]: `assertArraySubset` is deprecated. I *think* we have already replaced this, but even if not let's just drop this note so as not to mislead.

**Review by MGatner — CHANGES_REQUESTED** [review]

File: user_guide_src/source/testing/feature.rst (hunk 2 above)
Comment (paulbalandan) [reply user_guide_src/source/testing/feature.rst:384]: Yes. `assertJSONFragment` is no longer using phpunit's assertArraySubset. Better to drop that part.

Comment (kenjis) [reply user_guide_src/source/testing/feature.rst:384]: @MGatner done!

**Review by paulbalandan — APPROVED** [review]

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
Thank you @kenjis for the PR.
Thank you @MGatner @paulbalandan for the review.

**Pull request merged**